scrape/scrape.go

```diff
@@ -1566,6 +1601,13 @@ loop:
 			e = exemplar.Exemplar{} // reset for next time round loop
 		}
 
+		if shouldAppendMetadata {
+			_, metadataErr := app.AppendMetadata(ref, lset, meta)
+			if metadataErr != nil {
```

Comment:
Nit, we could do:

```suggestion
			if _, merr := app.AppendMetadata(ref, lset, meta); if merr != nil {
```
Same for exemplars.